### PR TITLE
Remove config.yaml

### PR DESCRIPTION
**Pull request by conorbros**
Removes the need for a config.yaml file

### CODE REVIEW

File: test-helpers/Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ license = "Apache-2.0"
 cassandra-cpp-driver-tests = ["cassandra-cpp"]
 
 [dependencies]
+clap = { version = "4.0.4", features = ["cargo", "derive"] }
```
Comment (rukai) [inline test-helpers/Cargo.toml:14]: what is this for?

**Comment by rukai** [comment]
this change alone wont be enough to resolve https://github.com/shotover/shotover-proxy/issues/993
The CLI args will still fail to parse before the tokio runtime is started.
So I dont think moving topology generation to runtime makes sense.

Instead I think we should have an error path that looks like:
```rust
fn Runner::new() -> Result<()> {
  if let Err(err) = Runner::new_inner() {
    create_trivial_tokio_runtime_just_for_tracing();
    error!("{:#?}", err.context("failed to init"));
    return Err(()); // or possibly just directly call exit(1)
  }
}
```

**Comment by conorbros** [comment]
I'll update this PR to just remove the config file.

**Comment by conorbros** [comment]
Hmm, will need to do a new release for this one. It breaks the docker example using the published DockerHub image.

**Pull request closed without merging**